src/mono/mono/mini/interp/transform.h

```diff
@@ -247,6 +258,9 @@ typedef struct
 	GPtrArray *seq_points;
 	InterpBasicBlock **offset_to_bb;
 	InterpBasicBlock *entry_bb, *cbb;
+	InterpBasicBlock **bblocks; // ordering of bblocks in reverse postorder dfs
+	int bblocks_count;
```

Comment:
What is difference between the `bblocks_count` and `bb_count`?

Reply:
bb_count is the global index that is used when allocating a new bblock. For a newly allocated bblock, it will result in an unique key.

bblocks_count is the number of bblock in `bblocks`. Here are stored the current relevant bblocks, less than 1bb_count` since bblocks can be merged, optimized out or unreachable. `dfs_index` is a key into this array.